src/teamwork.sh

```diff
@@ -32,21 +32,57 @@ teamwork::add_comment() {
   log::message "$response"
 }
 
+teamwork::add_tag() {
+  local -r tag_name=$1
+
+  if [ "$ENV" == "test" ]; then
+    log::message "Test - Simulate request. Task ID: $TEAMWORK_TASK_ID - Tag: ${tag_name//\"/}"
```

Comment:
This line is for testing, I would add to the message something like "Added tag".